### PR TITLE
Add refreshBlockVerifier method in ExchangeV3

### PR DESCRIPTION
**Pull request by kongliangzhong**

### CODE REVIEW

File: packages/loopring_v3/contracts/core/impl/ExchangeV3.sol [hunk 1]
```diff
@@ -111,6 +112,15 @@ contract ExchangeV3 is IExchangeV3, ReentrancyGuard
         state.agentRegistry = IAgentRegistry(_agentRegistry);
     }
 
+    function refreshBlockVerifier()
+        external
+        nonReentrant
+        onlyOwner
+    {
+        require(state.loopring.blockVerifierAddress() == address(0), "ZERO_ADDRESS");
```
Comment (Brechtpd) [inline packages/loopring_v3/contracts/core/impl/ExchangeV3.sol:120]: `require(state.loopring.blockVerifierAddress() != address(0), "ZERO_ADDRESS");`?

Comment (kongliangzhong) [reply packages/loopring_v3/contracts/core/impl/ExchangeV3.sol]: updated.

Comment (Brechtpd) [reply packages/loopring_v3/contracts/core/impl/ExchangeV3.sol]: Should be `require(state.loopring.blockVerifierAddress() != address(0), "ZERO_ADDRESS");`, `require(state.loopring.blockVerifierAddress() !== address(0), "ZERO_ADDRESS");` it too much javascript. :)

File: packages/loopring_v3/contracts/core/impl/ExchangeV3.sol [hunk 2]
```diff
@@ -111,6 +112,15 @@ contract ExchangeV3 is IExchangeV3, ReentrancyGuard
         state.agentRegistry = IAgentRegistry(_agentRegistry);
     }
 
+    function refreshBlockVerifier()
```
Comment (Brechtpd) [inline packages/loopring_v3/contracts/core/impl/ExchangeV3.sol:115]: Perhaps best to also add to `IExchangeV3`?

File: packages/loopring_v3/contracts/core/impl/ExchangeV3.sol (hunk 1 above)
Comment (kongliangzhong) [reply packages/loopring_v3/contracts/core/impl/ExchangeV3.sol]: Sorry.

File: packages/loopring_v3/contracts/core/impl/ExchangeV3.sol (hunk 2 above)
Comment (kongliangzhong) [reply packages/loopring_v3/contracts/core/impl/ExchangeV3.sol:115]: Okay.

Comment (kongliangzhong) [reply packages/loopring_v3/contracts/core/impl/ExchangeV3.sol:115]: updated, please take another look.

**Review by Brechtpd — APPROVED** [review]

**Pull request merged**